apps/video/test/marionette/lib/video.js

```diff
@@ -192,6 +197,58 @@ Video.prototype = {
   },
 
   /**
+   * Taps 'pause' button and checks that playback has stopped.
+   * returns false if the icon is not changed.
+   */
+  pausePlayback: function() {
+    this.client.helper.waitForElement(Video.Selector.playpauseButton).click();
+
+    // wait for the icon to change by checking for play icon
+    return this.client.helper.waitForElement(Video.Selector.playIcon)
+    .displayed();
+  },
+
+  /**
+   * Taps 'play' button and checks that the counter is progressing.
```

Comment:
so in this i have to wait for nothing is happening, then I would still need the async call ( where I wait for 2 seconds)  to wait arbitrary length of time and ensure nothing is changing, wouldn't I?  Tapforward() was easy because the time is incrementing, but in this case the time stays the same.. would setting interval: 10 make sure that the first check is done 10 milliseconds after, so it wouldn't pass trivially?